src/UI/TextField.elm

```diff
@@ -0,0 +1,354 @@
+module UI.TextField exposing
```

Comment:
Maybe we should implement this in another pr?

Reply:
Yep, moving to another PR...